### PR TITLE
Add some tests

### PR DESCRIPTION
**Pull request by alanhdu**
Companion PR to https://github.com/adicu/adi-website/pull/288.

This adds back all the tests that were left behind from the Great Split. I also went ahead and created a travis build and coverall job for the project (but you'll need to enable them on the upstream version.

[Build Status](https://travis-ci.org/alanhdu/eventum)
[Coverage Status](https://coveralls.io/github/alanhdu/eventum?branch=master)

### CODE REVIEW

**Comment by alanhdu** [comment]
Huh... looks like the tests actually passed w/ Pypy. Do you want to make that a feature of eventum (i.e. don't `allow_failure` for Pypy)?

File: eventum/authorize.py [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 import argparse
-from sys import argv
+import sys
 
 from oauth2client.file import Storage
 from oauth2client.client import flow_from_clientsecrets
 from oauth2client.tools import run_flow
 from oauth2client import tools
 
-if len(argv) < 3:
```
Comment (Howon) [inline eventum/authorize.py]: why not just use what we have before?

**Comment by Howon** [comment]
lgtm

**Comment by alanhdu** [comment]
Don't have write permissions, so someone will need to merge it in for me. (And also enable travis + coveralls on the main repo?)

**Pull request merged**